src/duplicate-search/sub-components/sort-select.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Dropdown, DropdownContent, DropdownItem, DropdownTrigger } from '../../common/components';
+import styles from '../duplicate-search-controls.module.css';
+import { IssueSortOption } from '../types';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { selectSort, setSearchParam, setSort } from '../duplicate-search-slice';
+import { ReactComponent as CheckIcon } from '../../common/svgs/check.svg';
+import { ReactComponent as DownIcon } from '../../common/svgs/chevron-down.svg';
+import { ReactComponent as SortIcon } from '../../common/svgs/sort.svg';
+import { Placement } from '@floating-ui/react';
+
+interface SortOptions {
+	label: string;
+	value: IssueSortOption;
+}
+
+export function SortSelect() {
+	const dispatch = useAppDispatch();
+	const currentSortOption = useAppSelector( selectSort );
+	const sortOptions: SortOptions[] = [
+		{
+			label: 'Relevance',
+			value: 'relevance',
+		},
+		{
+			label: 'Date added',
+			value: 'date-created',
+		},
+	];
+
+	const sortDropdownTriggerLabel =
+		sortOptions.find( ( sortOption ) => sortOption.value === currentSortOption )?.label || 'Sort';
+
+	const handleSortOptionClick = useCallback(
+		( sortOptionValue: string ) => {
```

Comment:
**suggestion (non-blocking):** I think it will be better and safer if we use `IssueSortOption` type as the parameter instead of string!  

I see that parameter is cast when passing to `setSort`, it'll be better to enforce the correct type in the beginning.

Reply:
Good call, got it fixed! 👍